### DISCUSSION TITLE
Welcome to ban-peers Discussions!

### DISCUSSION THREAD

**Discussion by SeaHOH** [SeaHOH/ban-peers#4] [General / 一般]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 欢迎！
  我们正在使用 Discussions 作为一般性交流讨论的地方：
  * 提出你的问题。
  * 分享你的意见和点子。
  * 与其他人进行交流。
  * 以及所有其它非 Issues 的东西，请畅所欲言 💪。

**Comment by xecutable** [comment]
I love this script however nowadays everyone either has some modification of a torrent program or a new one shows up. Can we convert the approach to a whitelist, instead of a blacklist. Seems much more reliable, as you can whitelist the legit clients, and completely ignore all of the fakers/modified ones without even knowing their names.

**Reply by SeaHOH** [reply]
I will not add whitelist mode, you may like to try [qBittorrent Enhanced Edition](https://github.com/c0re100/qBittorrent-Enhanced-Edition) and [Tixati](https://www.tixati.com/download).
